CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## Fixed
 
 - Error loading policy chain configuration JSON with null value [PR #626](https://github.com/3scale/apicast/pull/626)
+- Splitted `resolv.conf` in lines,to avoid commented lines  [PR #618](https://github.com/3scale/apicast/pull/618)
```

Comment:
Just a nitpic. Changelog should focus on "what" and not "how".
Here would be better fit: Skip lines starting with a comment when parsing `resolv.conf`

Reply:
I bear that in mind. Thanks